android/lib/ui-system/src/main/java/cz/quanti/rocketapp/android/lib/uisystem/system/ShowMessageInSnackBar.kt

```diff
@@ -0,0 +1,40 @@
+package cz.quanti.rocketapp.android.lib.uisystem.system
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import cz.quanti.rocketapp.android.lib.uisystem.presentation.UiScreenState
+
+/**
+ * Show message in snackbar.
+ * As per design rules, @Compose function should not result in "Nothing",
+ * so the caller is responsible for checking there is a message to show.
+ */
+@Composable
+fun UiScreenState.Data<*>.ShowMessageInSnackBar() {
```

Comment:
nevim, jestli je to idealni deleni, donutila me k tomu architektura - pouzivam zde localSnackbar ze system, a presentetion UiScreenState nemuze zaviset na system..